### DISCUSSION TITLE
Documentation error

### DISCUSSION THREAD

**Discussion by filippomenchini** [tidal-music/discussions#18] [Report bug]
Hi everyone!

First of all, I'm really happy about the new Tidal Developer program, congratulations!

I was playing around with the new APIs and I noticed an error in the Artist API reference, specifically the Get multiple artists endpoint.
The possible responses listed in the documentation are: 200, 400, 404, 405, 406, 415 and 500.
But, if you try to make an actual call to the API, the actual successful code is 207, which is correct given the fact that the API returns a mixture of responses.
Would it be possible to update the docs with the actual status code returned by the API?

Again, congratulations on this initiative, I'm really hyped to see the next updates!

**Comment by aliaksei-taliuk** [comment]
Hello @filippomenchini ! And welcome to our TIDAL Developer Forum.

Thanks a lot for your warm feedback. We will try to do our best to properly maintain what we already have, and open and add more features, and your participation and feedback are highly appreciated!

Meanwhile, this is a really good catch. The fix is already on its way.

Kind regards, Aliaksei

**Comment by aliaksei-taliuk** [comment]
@filippomenchini fixed, please let us know if you spot something. Closing this ticket.
